src/main.ts

```diff
@@ -4,6 +4,11 @@ import { MeetupAdapter } from "./meetup/MeetupAdapter";
 import { SimpleEventPublisher } from "./publisher/SimpleEventPublisher";
 import { DiscordAdapter } from "./discord/DiscordAdapter";
 
+function betterSetInterval(cb: () => void, interval: number) {
```

Comment:
Dodaj handling intervalów tutaj, niech runApplicatio bedzie jak najbardziej deklaratywny jak sie da

Reply:
I nazwij to jakos by mowilo co sie dzieje dokladnie.
W tym przypadku to pewnie setIntervalAndExecute

Reply:
Napisałem to na dworcu przed przyjazdem pociągu, myślałem że serio szybko trzeba łatkę wrzucić :P
Teraz jest ładniej :)